extensions/amqp/runtime/src/main/doc/limitations.adoc

```diff
@@ -0,0 +1 @@
+The JMS connection pooling isn't supported yet since there is still an open https://github.com/amqphub/quarkus-qpid-jms/issues/22[issue] with quarkus-qpid-jms.
```

Comment:
I think you need to run the build in the `amqp` extension module to regenerate the docs page.

Also an optional suggestion.

```suggestion
=== Connection Pooling

JMS connection pooling isn't supported yet since there is still an open https://github.com/amqphub/quarkus-qpid-jms/issues/22[issue] with quarkus-qpid-jms.
```

Reply:
Thanks @jamesnetherton !